### PR TITLE
[CF]-Azure-Event-Hub-Doc-Update

### PR DESCRIPTION
**Pull request by colefreeman**
# Description
This PR is an update to the Azure Event Hub documentation.

# How Has This Been Tested?
The document is appearing as expected in the mintlify dev env.

- [ ] Test A
- [ ] Test B

# Checklist
- [X] The PR is tagged with proper labels (bug, enhancement, feature, documentation)
- [X] I have performed a self-review of my own code
- [ ] I have added unit tests that prove my fix is effective or that my feature works
- [ ] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation

cc:
@johnson-mage 
Screenshot 2024-07-08 at 10 58 33 AM
Screenshot 2024-07-08 at 10 58 47 AM
Screenshot 2024-07-08 at 10 59 02 AM
Screenshot 2024-07-08 at 10 59 09 AM

### CODE REVIEW

File: docs/streaming/sources/azure-event-hub.mdx [hunk 1]
```diff
@@ -1,12 +1,91 @@
...
+1. Configure your io_config.yml file with the account variables below. If you need further information about how to store secrets in 
+Mage check out the [documentation](https://docs.mage.ai/production/deploying-to-cloud/secrets/Azure).
+
+```YAML
+  AZURE_CLIENT_ID: "{{ env_var('AZURE_CLIENT_ID') }}"
+  AZURE_CLIENT_SECRET: "{{ env_var('AZURE_CLIENT_SECRET') }}"
+  AZURE_STORAGE_ACCOUNT_NAME: "{{ env_var('AZURE_STORAGE_ACCOUNT_NAME') }}"
+  AZURE_TENANT_ID: "{{ env_var('AZURE_TENANT_ID') }}"
+```
+1. Add a data loader block to your pipeline and then select Azure Event Hub as the source.
```
Comment (johnson-mage) [inline docs/streaming/sources/azure-event-hub.mdx:26]: nit: quotes around "Azure Event Hub"

File: docs/streaming/sources/azure-event-hub.mdx [hunk 2]
```diff
@@ -1,12 +1,91 @@
...
+        messages: List of messages from the Event Hub.
+
+    Returns:
+        Transformed messages
+    """
+# Specify your transformation logic here
+    print(f'Transformed messages: {messages}')
+    return messages
+```
+5. Add a data exporter block and select “dummy”
```
Comment (johnson-mage) [inline docs/streaming/sources/azure-event-hub.mdx:64]: nit: capitalize "dummy"

File: docs/streaming/sources/azure-event-hub.mdx (hunk 1 above)
Comment (colefreeman) [reply docs/streaming/sources/azure-event-hub.mdx]: Made this change see picture 
Screenshot 2024-07-09 at 10 57 50 AM

File: docs/streaming/sources/azure-event-hub.mdx (hunk 2 above)
Comment (colefreeman) [reply docs/streaming/sources/azure-event-hub.mdx]: Made this change see picture 
Screenshot 2024-07-09 at 10 58 05 AM

**Review by johnson-mage — APPROVED** [review]

**Pull request merged**